### ISSUE TITLE
Using gradient checkpointing in RStudio/Keras?

### ISSUE THREAD

**Issue by donaldmusgrove** [rstudio/keras#419]
Is it possible to use gradient checkpointing with Keras in RStudio? If so, how? Here is the  [OpenAI repo](https://github.com/openai/gradient-checkpointing).

FYI, gradient checkpointing allows memory saving during model training, an issue I am currently facing with 3D segmentations. It is implemented for Tensorflow and Keras but I do not know how to use this within RStudio.
